### ISSUE TITLE
Improve lore2sphinx buildbot results for `projects/core/howto/telnet.xhtml`

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#4877]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)| @exarkun reported|
|-|-|
|Trac ID|trac#4877|
|Type|enhancement|
|Created|2011-02-11 05:43:17Z|
|Branch|https://github.com/twisted/twisted/tree/telnet-sphinx-4877|

like [#4566](https://github.com/twisted/twisted/issues/4566), except for the telnet howto

<details><summary>Searchable metadata</summary>

```
trac-id__4877 4877
type__enhancement enhancement
reporter__exarkun exarkun
priority__normal normal
milestone__Lore_to_Sphinx Lore to Sphinx
branch__branches_telnet_sphinx_4877 branches/telnet-sphinx-4877
branch_author__screwtape__jdb screwtape, jdb
status__closed closed
resolution__fixed fixed
component__core core
keywords__documentation documentation
time__1297402997000000 1297402997000000
changetime__1300074712000000 1300074712000000
version__None None
owner__washort washort

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_1"></a>|@exarkun commented|
|-|-|

(In [30687]) Branching to 'telnet-sphinx-4877'

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_2"></a>|@exarkun commented|
|-|-|

(In [30688]) Merge the telnet parts of the impossibly huge [[#4568](https://github.com/twisted/twisted/issues/4568)](https://github.com/twisted/twisted/issues/4568) branch

refs #4568
refs [#4877](https://github.com/twisted/twisted/issues/4877)

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="washort's avatar" src="https://avatars.githubusercontent.com/u/947487?s=50" width="50" height="50">](https://github.com/washort)|@washort set owner to @washort<br>@washort set status to `assigned`|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="washort's avatar" src="https://avatars.githubusercontent.com/u/947487?s=50" width="50" height="50">](https://github.com/washort)<a name="note_7"></a>|@washort commented|
|-|-|

Output looks right on Sphinx now and processes on Lore. Merging.

**Comment by twisted-trac** [comment]
|[<img alt="washort's avatar" src="https://avatars.githubusercontent.com/u/947487?s=50" width="50" height="50">](https://github.com/washort)<a name="note_8"></a>|@washort set status to `closed`|
|-|-|

(In [30999]) Merge telnet.xhtml changes from telnet-sphinx-4877.

Author: screwtape, jdb
Reviewer: washort
Fixes: [#4877](https://github.com/twisted/twisted/issues/4877)

Fixes formatting changes needed to make sphinx happy with telnet.xhtml.
